### PR TITLE
Added Missing Object Templates for Imperial Restuss Commendations Weapons.

### PR DESCRIPTION
**Pull request by talisaknight**
…pons.

### CODE REVIEW

**Review by BloodCommando — COMMENTED** [review]
It may look like there would be Imperial Restuss weapons, but in fact the "Rebel" ones are shared by both sides. The file names you're referencing wp_pistol_imperial etc are actually the weapons granted when reaching the GCW rank of General. Those weapons are available to both sides as well. This is a case of SOE going with the naming convention of the models, versus where they actually end up in game.

In short, Restuss weapons consist of "mandalorian" and "rebel" weapons."Imperial" weapons are the GCW General rank weapons. This commit is essentially re-adding the GCW rank General weapons again to objects and the master_item.

**Review by AconiteX — CHANGES_REQUESTED** [review]
Changes requested pursuant to Blood's comments.

**Comment by talisaknight** [comment]
Closing.
